app/models/concerns/foreman_bootdisk/orchestration/compute.rb

```diff
@@ -71,10 +82,10 @@ def setAttachIsoImage
         failure format(_('Failed to attach ISO image to CDROM drive of instance %{name}: %{message}'), name: name, message: e.message), e
       end
 
-      def delAttachIsoImage; end
```

Comment:
I believe this needs to stay there or a rollback action won't work in case an error occurs.

Reply:
done